app/views/static_pages/home.html.erb

```diff
@@ -0,0 +1,7 @@
+<% provide(:title, "Home") %>
+<h1>Sample App</h1>
```

Comment:
Take a look at https://guides.rubyonrails.org/i18n.html we use for internationalising stings